### DISCUSSION TITLE
Repositories description doesn't take full-width on mobile website.

### DISCUSSION THREAD

**Discussion by fabiocaccamo** [community/community#32880] [Other Feature Feedback, Questions, & Ideas]
Repositories layout (description) doesn't take full-width on mobile website in the repositories section:

<img src="https://user-images.githubusercontent.com/1035294/189843544-7de3c6e2-5596-48a9-a87f-32e0f35faed6.PNG" width="400" />

**Comment by stevepopovich** [comment]
This category is for feedback on the native Mobile app. Changing categories!

**Reply by fabiocaccamo** [reply]
Sorry for the misunderstanding.

The "general" category is too generic, wouldn't it be better to add a "Website" category?

**Reply by stevepopovich** [reply]
Maybe so! All the of the categories apply to the website, generally. Except mobile 🙃

**Comment by iSbellaJack1** [comment]
There is a chance that it is true! There are a number of categories that apply to the website in general. With the exception of mobile devices.
Or Someone else like [SSGC](https://ssgc.pk/)
